packages/libs/react-ui/src/components/Accordion/Accordion.css.ts

```diff
@@ -13,32 +16,49 @@ export const accordionSectionClass = style([
     },
   },
 ]);
-export const accordionTitleClass = style([
-  sprinkles({
-    display: 'flex',
+
+export const accordionSectionHeadingClass = style([
```

Comment:
Could some of these classes be consolidated into one now that they are all part of the same button?